Python/optimizer.c

```diff
@@ -643,14 +643,13 @@ translate_bytecode_to_trace(
                 int bitcount = _Py_popcount32(counter);
                 int jump_likely = bitcount > 8;
                 /* If bitcount is 8 (half the jumps were taken), adjust confidence by 50%.
-                   If it's 16 or 0 (all or none were taken), adjust by 10%
-                   (since the future is still somewhat uncertain).
+                   If it's 16 or 0 (all or none were taken), adjust by 0%.
```

Comment:
It seems odd to have a comment saying "adjust by 0%". Maybe just delete this comment?